doc/source/gitwash/development_workflow.rst

```diff
@@ -78,7 +78,7 @@ what the changes in the branch are for.  For example ``add-ability-to-fly``, or
     git fetch upstream
     # Make new feature branch starting at current trunk
     git branch my-new-feature upstream/main
-    git checkout my-new-feature
+    git switch my-new-feature
```

Comment:
This won't work, it needs: `git switch -n my-new-feature`.

Below, the branch name `cool-new-feature` is used; should we use the same one?

Reply:
ThankYou @stefanv I think this would be a great idea. I am changing the name of the branch and using the changes suggested by you.